server/src/api/api.graphql

```diff
@@ -78,3 +84,54 @@ type Category {
     # The human-readable, user-facing name for this category
     category_name: String!
 }
+
+enum RequestStatus {
+	SUBMITTED
+	APPROVED
+	DENIED
+	ABANDONED
+    CANCELLED
+	READY_FOR_PICKUP
+	FULFILLED
+	RETURNED
+	LOST
+	DAMAGED
+}
+
+type Request {
+	request_id: Int!,
+	user: User!,
+	item: Item!,
+	status: RequestStatus!,
```

Comment:
what's the plan for partial fulfillment? complete denial?

Reply:
Latest thought was to change the quantity, perhaps do something like "split" the request into one that's denied and one that's approved.  Was thinking we'd need the same type of thing for handling partially lost requests (split into 1 returned and 1 lost/damaged)